bundles/org.openhab.binding.vektiva/src/main/resources/ESH-INF/thing/smarwi.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vektiva"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Smarwi Thing -->
+	<thing-type id="smarwi">
+		<label>Vektiva Smarwi Thing</label>
+		<description>Smarwi thing for Vektiva Binding</description>
+
+		<channels>
+			<channel id="control" typeId="control" />
+			<channel id="status" typeId="status" />
+		</channels>
+
+		<config-description>
+			<parameter name="ip" type="text" required="true">
+				<label>Smarwi IP address</label>
+				<context>network-address</context>
+				<description>IP address of the Smarwi thing on the local network.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="true" min="5">
```

Comment:
If you have a default, it's not `required`. You can also apply the default in your config-class, then you are sure that it is not null.

Reply:
makes sense, will fix it.